docker-compose.yml

```diff
@@ -185,13 +180,9 @@ services:
       - prometheus
       - compliance-ssg
     environment:
-      - DATABASE_SERVICE_NAME=postgres
-      - POSTGRES_SERVICE_HOST=db
-      - POSTGRESQL_DATABASE=compliance_dev
       - POSTGRESQL_TEST_DATABASE=compliance_test
-      - POSTGRESQL_USER=insights
-      - POSTGRESQL_PASSWORD=insights
       - SETTINGS__PROMETHEUS_EXPORTER_HOST=prometheus
```

Comment:
This should come from devel.json

Reply:
they come from it, I removed from the file